### DISCUSSION TITLE
Expo 48+ with `@expo/next-adapter`

### DISCUSSION THREAD

**Discussion by JWesorick** [nandorojo/solito#382] [Q&A]
The latest `@expo/next-adapter` sets `"expo": "^46",` as a peer dependency. This prevents be using expo 48 in the expo project. Even though `@expo/next-adapter` is only used in the next project there is the shared lock file between the two apps. 

Am I doing something wrong? What's the best way to go about using expo 48?

**Comment by JWesorick** [comment]
Added

```json
 "overrides": {
    "expo": "^48"
  }
```

to the root `package.json` for now. We will see if this causes issues later.

**Reply by nandorojo** [reply]
are you using yarn?

**Reply by JWesorick** [reply]
Yea I was not. I am now.
